_includes/datagov-search-display.html

```diff
@@ -69,10 +69,15 @@
   {{- content -}}
 </div> 
 <div class="search pagination padding--bottom--xl">
-  <span class="sgds-icon sgds-icon-arrow-left is-size-4"></span>
+  <span class="bx bx-chevrons-left is-size-4"></span>
```

Comment:
nit: the left and right chevrons do not appear as a clickable element to the user on desktop (cursor is an arrow instead of a hand). We should show the hand instead.

Reply:
https://github.com/isomerpages/isomerpages-template/pull/320/commits/4bd7d318b8be85d638a59cf44dea093d571a258c